src/Scripty.CustomTool/BaseCodeGenerator.cs

```diff
@@ -1,4 +1,6 @@
-﻿
+﻿//Dave Glick - is this just template copyright stuff that was brought in or 
+// is this something else?
+
```

Comment:
Some of these classes are included in the newer VS shell references. You may be able to remove them if you up the shell ref's.